test/e2e_flags.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package test
+
+import (
+	"flag"
+	"log"
+)
+
+// Flags holds the command line flags specific to knative-gcp.
+var Flags EnvironmentFlags
+
+// EventingEnvironmentFlags holds the e2e flags needed only by the eventing repo.
+type EnvironmentFlags struct {
+	WorkloadIdentityEnabled bool
+	PubsubServiceAccount    string
+}
+
+// InitializeFlags registers flags used by e2e tests, calling flag.Parse() here would fail in
+// go1.13+, see https://github.com/knative/test-infra/issues/1329 for details
+func InitializeFlags() {
+	flag.BoolVar(&Flags.WorkloadIdentityEnabled, "workloadIdentityEnabled", false, "Indicating whether the workload identity is enabled or not.")
```

Comment:
My preference would be to name this `workloadIndentity`, as I feel 'enabled' is redundant with the value of the flag.

Reply:
@chizhg mentioned to change the name (https://github.com/google/knative-gcp/pull/709#discussion_r398316896) from` workloadIndentity` to `WorkloadIdentityEnabled `, that is for the name after parsing the flag to the e2e-test. But I changed all, for flag and for the name after parsing it. Do you think we need to use two different names for this, or `workloadIndentity` can be used in both place?